src/core_modules/capture-core/components/Pages/Search/SearchForm/SearchForm.epics.js

```diff
@@ -12,75 +12,116 @@ import { actionCreator } from '../../../../actions/actions.utils';
 
 const trackerCaptureAppUrl = () => (process.env.REACT_APP_TRACKER_CAPTURE_APP_PATH || '..').replace(/\/$/, '');
 
-export const onScopeProgramFindUsingUniqueIdentifierEpic = (action$: InputObservable, store: ReduxStore) =>
+const filtersForUniqueIdSearchQuery = (searchTerm) => {
```

Comment:
Here you will find here 4 epics. 

1. for the search via unique id AND scope `Program`
2. for the search via unique id AND scope `TrackedEntityType`
3. for the search via attributes AND scope `Program`
4. for the search via attributes AND scope `TrackedEntityType`

I am also doing the refactoring I mentioned here https://github.com/dhis2/capture-app/pull/822/files#r450698667

Reply:
I like the separation 👍